README.md

```diff
@@ -35,25 +35,32 @@ const Koa = require('koa')
 const app = new Koa()
 
 const router = require('koa-router')()
-const WebMonetization = require('koa-web-monetization')
-const monetization = new WebMonetization()
-
+const { WebMonetizationMiddleWare, KoaWebMonetization } = require('koa-web-monetization')
+const monetizer = new KoaWebMonetization()
+const EXPRESS_WEB_MONETIZATION_CLIENT_PATH =  '/node_modules/express-web-monetization/client.js'
```

Comment:
should use `path.resolve(path.dirname(require.resolve('koa-web-monetization')), 'client.js')` instead, so that it works even when this module is a dependency of another. Although it might be nice to expose a standard function to do this for you.

Reply:
Super neat!